docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -246,3 +249,211 @@ Then server `$hostname` supports secure protocols that $rule `$protocols`
 ----
 Then server `vividus-test-site.herokuapp.com` supports secure protocols that contain `TLSv1.2,TLSv1.3`
 ----
+
+=== Verify JSON context contains data by JSON path
+
+Checks if the JSON context contains an expected data by a JSON path
+
+[source,gherkin]
+----
+Then JSON element by JSON path `$jsonPath` is equal to `$expectedData`$options
+----
+
+* `jsonPath` - {json-path}
+* `expectedData` - expected value of found element by JSON path
```

Comment:
`expected JSON`

Reply:
it can be string, number, object or array